examples/src/main/r/ml.R

```diff
@@ -25,8 +25,9 @@ library(SparkR)
 sc <- sparkR.init(appName="SparkR-ML-example")
 sqlContext <- sparkRSQL.init(sc)
 
-############################ spark.glm and glm ##############################################
 
+############################ spark.glm and glm ##############################################
+# $example on$
```

Comment:
Move this line to L28, it's better we can include the annotation line ```spark.glm and glm``` which can help users to understand corresponding code.